transformer_engine/pytorch/distributed.py

```diff
@@ -615,7 +615,7 @@ def checkpoint(
             "`get_rng_state_tracker` must be passed as keyword arguments to `checkpoint`.",
             DeprecationWarning, stacklevel=2,
         )
-        distribute_saved_activations, get_rng_state_tracker, tp_group = args[:3] # pylint: disable=unbalanced-tuple-unpacking
+        distribute_saved_activations, get_rng_state_tracker, tp_group = args[:3]
```

Comment:
I feel like it's better to split up a one-liner than to remove a Pylint warning:
```suggestion
        distribute_saved_activations = args[0]
        get_rng_state_tracker = args[1]
        tp_group = args[2]
```